tcf_website/models/models.py

```diff
@@ -213,6 +207,21 @@ def average_difficulty(self):
             instructor=self).aggregate(
             models.Avg('difficulty'))['difficulty__avg']
 
+    def get_courses(self):
+        """Gets all Courses taught by a given Instructor"""
+        # More specifically, all Courses where this Instructor has taught a Section
+        # Might be good to store this data as a many-to-many field in
+        # Instructor instead of computing?
+        course_ids = list(
+            Section.objects.filter(
+                instructors=self).distinct().values_list(
+                'course_id', flat=True))
+        # TODO: remove old courses that haven't been taught in 5+ years;
```

Comment:
I'm not sure what I was thinking here....I think the idea was that 5+ years is the scope of relevancy for a review (because things probably change), but tCF 1 just keeps all review data. I think it's pretty safe to leave out anything with a 3-digit course number because I highly doubt UVA will ever go back to that, and manually figuring out what got renamed is probably a wasted effort considering how few reviews are likely from that time anyways.

I'll just remove that comment line, though.